ontology/account/account.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, private_key: str or bytes, scheme=SignatureScheme.SHA256withE
 
     def generate_signature(self, msg: bytes) -> object:
         """
-        Generate and return verified message signed with key pair.
+        Generate verified message signed with key pair.
```

Comment:
Generate message signature in account's signature scheme.

Reply:
Happy accident. fixed.